### PR TITLE
fix(no-gas): add click 'here' link

### LINKED ISSUE #265: “Not Enough Gas": Update Transaction Message Display & UX
The error message that appears when the wallet user does not have enough gas to send tokens is incorrect and unhelpful. The phrasing of the message may also cause distress to a user unfamiliar with how ERC-20 tokens work.

Wireframe: https://balsamiq.cloud/sv8gyk0/piks2o2/r0329

Key Functionality:
1) User can get notified in a friendly way that they don't have enough gas (ETH) to pay for this transaction. 
2) Display this message across ETH, KEY, and ERC-20 sending transaction workflows.

Dependency:
Please prioritize Issue #312 first as the displayed network transaction fee will be pulled from ETH Gas Station. We can then compare this to the current wallet balance and trigger this message earlier in the workflow for a better UX.

### PR DESCRIPTION
**Pull request by sk91**
fixes #265 

related: https://github.com/SelfKeyFoundation/selfkey-ui/pull/17

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,7 +108,7 @@
     "redux-thunk": "^2.3.0",
     "request": "2.87.0",
     "reselect": "^3.0.1",
-    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git",
+    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git#fix/no-gas",
```
Comment (dfcowell) [inline package.json]: Prefer using commit hashes. This will fail if that branch gets merged and deleted.

Comment (sk91) [reply package.json]: @dfcowell yes, I usually update after selfkey-ui was merged. The hash can change after merge no?
Will update now

**Comment by sk91** [comment]
@dfcowell fixed

**Comment by rhdelima** [comment]
Passed on MAC. waiting for a WIN build

**Comment by rhdelima** [comment]
Passed QA

**Pull request merged**